tests/installation/partitioning/guided_setup_multiple_disks.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The test module performs guided partitioning
```

Comment:
we could also just name it 'guided_setup.pm' for now.

Reply:
Yes, that's what I wanted to ask you, if we are aiming data driven test module, or I should introduce the method which aims multidisk installations only. So yes, let's call it `guided_setup`.